report/unsafe/sections/albin_arrowhead_communication.tex

```diff
@@ -0,0 +1,92 @@
+\subsection{Arrowhead framework}
+Arrowhead framework is a framework designed to aid in the functonality of Iot-devices, havning a focus on large scale industry and
+productivity. The framework provides a number of core system, systems that are required to run to enable certain functionality. This 
+project uses five of them nameley, service registry, orchestrator, authorization, gateway and gatekeeper. The service registry is used 
+to register systems and services, a system is defined as either a provider or a consumer. A provider provides a service for examample 
+the reading of a temperature sensor and in turn the consumer consumes that service which in this example might be to log the temperature
+in a database. A service is therefore what the provider is providing and the consumer consuming, more on this later. The orchestrator is
+used to decide which consumer and acces which provider and what services said provider is entitled to provide the conusmer with. The
+authorization is used to declare which providers are authozired within both the local cloud, intra cloud authorization, and between
+differnt clouds, inter cloud authorization. When defining inter or intra cloud rules to user also has to decide which interface should 
```

Comment:
"differnt clouds, inter cloud authorization"

is it suppose to be and instead of comma or is it a general term for all the above.